ViAn/Project/videoproject.cpp

```diff
@@ -120,7 +120,7 @@ void VideoProject::write(QJsonObject& json){
     QJsonArray json_analyses;
     for(auto it2 = analyses.begin(); it2 != analyses.end(); it2++){
```

Comment:
also can't all these loops also be written as the "for (POI poi : pois)" kind or is there a special reason the iterator loop is used instead?